lib/AST/ASTPrinter.cpp

```diff
@@ -3433,7 +3433,9 @@ void PrintAST::visitAccessorDecl(AccessorDecl *decl) {
   printAttributes(decl);
   // Explicitly print 'mutating' and 'nonmutating' if needed.
   printMutabilityModifiersIfNeeded(decl);
-
+  if (decl->isConsuming()) {
+    Printer.printKeyword("__consuming", Options, " ");
+  }
```

Comment:
Do you know why it's not already printed by `printAttributes`?

Reply:
It seems we exclude contextual decl attributes specifically [here](https://github.com/apple/swift/blob/main/lib/AST/ASTPrinter.cpp#L1111).